awx/main/tests/docs/test_swagger_generation.py

```diff
@@ -176,4 +176,8 @@ def teardown_class(cls):
                 r'\1xxxx\3',
                 data
             )
+            data = re.sub(
+                r'"action_node": "awx-[^"]+"',
```

Comment:
zuul generates randomized hostnames for the containers, so this was making example activity stream output non-deterministic